spring-pulsar/src/main/java/org/springframework/pulsar/listener/ConcurrentPulsarMessageListenerContainer.java

```diff
@@ -108,6 +108,7 @@ private void configureChildContainer(int index, DefaultPulsarMessageListenerCont
 			this.executors.add(exec);
 			container.getContainerProperties().setConsumerTaskExecutor(exec);
 		}
+		container.setNegativeAckRedeliveryBackoff(this.negativeAckRedeliveryBackoff);
```

Comment:
Should we add some light coverage for this in the corresponding CPMLCT? Just that it got set on the child container would be a sufficient check IMO.

Reply:
That will be good. Let me look into it.